src/logical/permutator.rs

```diff
@@ -99,8 +99,8 @@ impl Permutator {
 
     /// Creates a [`Permutator`] based on one a list of [`VecT`], with a given offset.
     ///
-    /// The sorting of values is done by taking the first [`VecT`]] and sort according to these values.
-    /// If two elements are the equal to each other, the next [`VecT`] will be taken to check if the comparison is different.
+    /// The sorting of values is done by taking the first [`VecT`] and sort according to these values.
+    /// If two elements are the equal to each other, the next [`VecT`] is taken to check if the comparison is different.
     /// If all [`VecT`] comparisons result in equality, the original order of the two values is preserved.
```

Comment:
This needlessly duplicates the doc from above – describing the sorting behaviour should probably go on the `struct`.

Reply:
It depends on the construction method on how the sorting behaviour will be. The Permutator itself only allows to change the ordering.

Reply:
It's still _describing the same behaviour_, since `sort_from_multiple_vec` simply calls `sort_from_multiple_vec_with_offset`, so the description should be there only once, with a reference to wherever it is. We _most definitely_ want to have two incongruent descriptions of the same behaviour, as we have now.